### PR TITLE
Curvefs tools: readme

### LINKED ISSUE #764: curvefs_tools: need some manuals for the tools
**Is your feature request related to a problem? (你需要的功能是否与某个问题有关?)**

curvefs_tool has no manual

**Describe the solution you'd like (描述你期望的解决方法)**

add README.md in curvefs/src/tools/README.md

**Describe alternatives you've considered (描述你想到的折衷方案)**

**Additional context/screenshots (更多上下文/截图)**

### PR DESCRIPTION
**Pull request by Cyber-SiKu**
1. add curvefs_tool readme
2. adjust some curvefs_tool output
3. fix check-copyset no output by set show = true
4. fix query-partition name show error in --example

### What problem does this PR solve?

Issue Number: close #764  

Problem Summary:

1. no manual
2. curvefs_tool help outout
3. check-copyset no output
4. querypartition name error

### What is changed and how it works?

1. add curvefs_tool readme
2. adjust some curvefs_tool output
3. fix check-copyset no output by set show = true
4. fix query-partition name show error in --example

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: curvefs/src/tools/README.md [hunk 1]
```diff
@@ -0,0 +1,501 @@
...
+curvefs_tool version
+```
+
+Output:
+
+```shell
+1.0.0
+```
+
+[Back to Toc](#Contents)
```
Comment (SeanHai) [inline curvefs/src/tools/README.md:62]: TOC

File: curvefs/src/tools/README.md [hunk 2]
```diff
@@ -0,0 +1,501 @@
...
+## usage
+
+### **usage-metadata**
+
+show the metadata usage of cluster
+
+Usage：
+
+```shell
+curvefs_tool delete-fs
```
Comment (SeanHai) [inline curvefs/src/tools/README.md:291]: curvefs_tool usage-metadata

File: curvefs/src/tools/README.md [hunk 3]
```diff
@@ -0,0 +1,501 @@
...
+## check
+
+### **check-copyset**
+
+checkout copyset status
+
+Usage:
+
+```shell
+curvefs_tool checkout-copyset -copysetId=10 -poolId=1
```
Comment (SeanHai) [inline curvefs/src/tools/README.md:341]: Must specify copysetId and poolId ? Can check all copysets at a time?

File: curvefs/src/tools/README.md [hunk 4]
```diff
@@ -0,0 +1,501 @@
...
+```
+
+### **list-coysetInfo**
+
+list all copysetInfo in cluster
+
+Usage:
+
+```shell
+curvefs_tool list-copysetInfo
```
Comment (SeanHai) [inline curvefs/src/tools/README.md:200]: Maybe list all the copysets on a metaserver is needed.

List command also need to add 'list server; list metaserver; list partition; list partiton of fs and so on'.

File: curvefs/src/tools/README.md [hunk 5]
```diff
@@ -0,0 +1,501 @@
+# curvefs_tool
+
+Tools for Curve filesystem.
```
[the remark this answers is not in the document]
Comment (aspirer) [reply curvefs/src/tools/README.md:3]: CurveFS

File: curvefs/src/tools/list/curvefs_copysetinfo_list.cpp [hunk 6]
```diff
@@ -62,7 +62,11 @@ bool CopysetInfoListTool::AfterSendRequestToHost(const std::string& host) {
         ret = false;
     } else if (show_) {
         for (auto const& i : response_->copysetvalues()) {
-            std::cout << i.DebugString() << " ";
+            std::cout << "copyset["
+                      << copyset::GetCopysetKey(i.copysetinfo().poolid(),
+                                                i.copysetinfo().copysetid())
+                      << "]:" << std::endl;
+            std::cout << i.DebugString() << std::endl;
```
[the remark this answers is not in the document]
Comment (Cyber-SiKu) [reply curvefs/src/tools/list/curvefs_copysetinfo_list.cpp:69]: fix

File: curvefs/src/tools/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (Cyber-SiKu) [reply curvefs/src/tools/README.md:3]: > 

fix

File: curvefs/src/tools/README.md (hunk 1 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/README.md:62]: > 

fix

File: curvefs/src/tools/README.md (hunk 2 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/README.md:291]: > 

fix

File: curvefs/src/tools/README.md (hunk 3 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/README.md:341]: > 
poolId is necessary for request.
stautus-copyset can check all copysets at a time

File: curvefs/src/tools/README.md (hunk 4 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/README.md:200]: > 

will add.

**Review by SeanHai — APPROVED** [review]

File: curvefs/src/tools/README.md [hunk 7]
```diff
@@ -0,0 +1,501 @@
...
+-status:
+status: COPYSET_OP_STATUS_SUCCESS copysetStatus { state: 1 peer { address: "10.182.2.45:16701:0" } leader { address: "10.182.2.45:16701:0" } readonly: false term: 2 committedIndex: 1 knownAppliedIndex: 1 pendingIndex: 0 pendingQueueSize: 0 applyingIndex: 0 firstIndex: 1 lastIndex: 1 diskIndex: 1 epoch: 0 }
+status: COPYSET_OP_STATUS_SUCCESS copysetStatus { state: 4 peer { address: "10.182.2.45:26701:0" } leader { address: "10.182.2.45:16701:0" } readonly: false term: 2 committedIndex: 1 knownAppliedIndex: 1 pendingIndex: 0 pendingQueueSize: 0 applyingIndex: 0 firstIndex: 1 lastIndex: 1 diskIndex: 1 epoch: 0 }
+status: COPYSET_OP_STATUS_SUCCESS copysetStatus { state: 4 peer { address: "10.182.2.45:36701:0" } leader { address: "10.182.2.45:16701:0" } readonly: false term: 2 committedIndex: 1 knownAppliedIndex: 1 pendingIndex: 0 pendingQueueSize: 0 applyingIndex: 0 firstIndex: 1 lastIndex: 1 diskIndex: 1 epoch: 0 }
+
+...
+
+```
+
+[TOC](#Contents)
```
Comment (Wine93) [inline curvefs/src/tools/README.md:150]: It maybe not work.

Comment (Cyber-SiKu) [reply curvefs/src/tools/README.md:150]: > It maybe not work.

fix

**Review by Wine93 — APPROVED** [review]

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Pull request merged**